web/client-api/src/main/java/io/deephaven/web/public/linked/child.html

```diff
@@ -26,7 +25,10 @@ <h3>Outgoing link summary</h3>
 
 </ul>
 
-<script>
+<script type="module">
+    import {dh} from './dh-core.js';
```

Comment:
I think this needs `../`
```suggestion
    import {dh} from '../dh-core.js';
```

Reply:
Also fix the whitespace to be consistent